src/views/Home.vue

```diff
@@ -121,7 +103,7 @@ export default {
     const remotePeerId = this.edgePeerId
 
     if (setupStatus || remotePeerId) {
-      this.hasSetupSystem = true
+      this.$router.push('timeline')
     }
```

Comment:
Please add test coverage.

Reply:
I would check how to cover this in a test suite, it would require me tapping into vuex from the test suite but I should be able to do it.

Reply:
@vickywane any luck figuring out the warning below?
`Variable 'on' is already declared in the upper scope`

Reply:
Sorry, this skipped my focus. 

I plan fixing them in a small separate PR before my next PR